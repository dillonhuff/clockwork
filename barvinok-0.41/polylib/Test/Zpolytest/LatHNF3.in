# HNF of this Matrix (not a lattice)
M 1
5 5
0 0 0 1 0
4 0 4 0 -4
4 0 4 0 -4
4 0 4 0 -4
0 0 0  0 1
D 0
F 2

