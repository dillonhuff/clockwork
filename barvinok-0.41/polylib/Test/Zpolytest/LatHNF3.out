5 5
   1   0   0   0   0
   0   4   0   0   0
   0   4   0   0   0
   0   4   0   0   0
   0   0   0   0   1
5 5
   0   0   0   1   0
   1   0   1   0  -1
   0   1   0   0   0
   1   0   0   0   0
   0   0   0   0   1
